superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -136,16 +138,13 @@ def get_recipe_text(self, distributor, license_text, die_msg=None):
         elif isinstance(self.license, list):
             self.license = self.license[0].replace(' ', '-')
             ret += 'LICENSE = "' + self.license + '"\n'
-            """
-            TODO(allenh1): add this functionality
```

Comment:
This functionality should use `' '.join(self.license)` to be more pythonic and compact.

Reply:
Thank you for the feedback! I'll implement that, and check out rebasing rn :+1: